app/notify_admin.go

```diff
@@ -217,7 +258,25 @@ func (a *App) FinishSendAdminNotifyPost(trial bool, now int64) {
 		mlog.Error("Unable to finish send admin notify post job", mlog.Err(err))
 	}
 
-	// all the notifications are now sent in a post and can safely be removed
+	// all the requested features notifications are now sent in a post and can safely be removed except
+	// the plugin notify admin. we keep it as we do not want the same user to send the notification for the same plugin
+	// from the work template feature. We update the NotifyAdmin sentAt to keep track of it.
+	if len(pluginBasedData) > 0 {
```

Comment:
Because we don't need the early return, and because code within a range block doesn't execute if the iterable is empty, we don't need the top level `if` here. I'd recommend removing for simplicity.